src/main/python/hydra/lib/hydrabase.py

```diff
@@ -46,14 +46,15 @@ class AppGroup(object):
     group_name:     group name
     analyser:       Analyser class "name" e-g HAnalyser not HAnalyser()
     """
-    def __init__(self, hydra, app_name, group_name, analyser=None):
+    def __init__(self, hydra, app_name, group_name, num_app_instances, analyser=None):
         self.group_info = {}
         self.hydra = hydra
         if not analyser:
             raise Exception("AppGroup needs analyser class name passed as a name, curr val = %s" % analyser)
         self.analyser = analyser
         self.group_name = group_name
         self.app_name = app_name
+        self.num_app_instances = num_app_instances
```

Comment:
If it's being used by kraken than kraken can create it, it does not belong here.